evap/student/templates/student_vote.html

```diff
@@ -51,7 +51,25 @@
             </small>
         </div>
     {% endif %}
-    <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name }})</h3>
+    <h3 class="mb-3">
+        {{ evaluation.full_name }} ({{ evaluation.course.semester.name }})
+
+        <div class="btn-switch btn-switch-light float-end">
+            <div class="btn-switch-label my-auto">{% translate "Questionnaire language" %}</div>
+            <div class="btn-switch btn-group">
+                {% for language_code, language_name in languages %}
+                    <a data-bs-toggle="tooltip"
+                       data-bs-placement="bottom"
+                       title="{{ language_name }}"
+                       role="button"
+                       href="{% url 'student:vote' evaluation.pk %}?language={{ language_code }}"
+                       class="btn btn-sm btn-light{% if evaluation_language == language_code %} active{% endif %}">
+                        <span id="span-set-questionaire-language-{{ language_code }}">{{ language_code|upper }}</span>
```

Comment:
The id here seems to be unused. For the language picker in the nav bar, we use such ids to trigger a spinning icon until the page reloads. The code to add this icon seems to be missing